ui/components/component-library/avatar-token/avatar-token.stories.js

```diff
@@ -93,18 +116,18 @@ export const ColorBackgroundColorAndBorderColor = (args) => (
       {...args}
       backgroundColor={COLORS.GOERLI}
       borderColor={COLORS.GOERLI}
-      tokenName="G"
+      name="G"
       color={COLORS.PRIMARY_INVERSE} // TODO: This will have to be added to the AvatarBase component as a prop so we can change the color of the text and to the base avatar
```

Comment:
I think this TODO is no longer needed but I have identified an issue with our colors. Could update this TODO to say update to `COLORS.GOERLI_INVERSE`